core/src/test/scala/org/apache/spark/sql/delta/DeltaCDCStreamSuite.scala

```diff
@@ -1006,6 +1006,40 @@ trait DeltaCDCStreamSuiteBase extends StreamTest with DeltaSQLCommandTest
       }
     }
   }
+
+  // LC-1281: Ensure that when we would split batches into one file at a time, we still produce
+  // correct CDF even in cases where the CDF may need to compare multiple file actions from the
+  // same commit to be correct, such as with persistent deletion vectors.
+  test("double delete-only on the same file") {
+    withTempDir { tableDir =>
+      val tablePath = tableDir.toString
+      spark.range(start = 0L, end = 10L, step = 1L, numPartitions = 1).toDF("id")
```

Comment:
Why `toDF("id")`?

Reply:
Just making the name of column explicit, rather than having to know what `spark.range` happens to name its column.